### PR TITLE
Square Root Added #920 Solved with Readme

### PR DESCRIPTION
**Pull request by krishna-NIT**
#920 Done in Java
@nakul-19 @rudrakshi99 @OjusWiZard Please Review it All Changes are Done.

### CODE REVIEW

**Comment by krishna-NIT** [comment]
@yasharth291 Please review it because Daily I get Conflicts in Readme File, I have to Resolve them Daily, Am I making any Mistake in Readme ?, can you please tell me why I am Getting Conflicts in Readme always, coz I am new to GIT and GITHUB

**Comment by krishna-NIT** [comment]
@yasharth291 will you please review it !

**Comment by krishna-NIT** [comment]
@Rohan27s @tarun26091999  Please Review it and Let me Know if any changes

File: Code/Java/sqrt.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}
+/*
+    Test Cases:
+        Input: 9
+        Output: 3
+
+        Input: 8
+        Output: 2
+
+        Time Complexity: O(log n)
```
Comment (Rohan27s) [inline Code/Java/sqrt.java:51]: mention the value of n here

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by krishna-NIT** [comment]
@Rohan27s Done Sir!

**Comment by krishna-NIT** [comment]
@rish-singhal Last remaning sir

**Pull request merged**